MediaWikiTelegramBot.js

```diff
@@ -65,53 +80,71 @@ tgBot.onText( /^\/setlanguage ?(.*)/, function ( msg, match ) {
     debug( userID, "newLanguageCode is " + newLanguageCode );
     debug( userID, "newLanguageCode type is " + typeof( newLanguageCode ) );
 
+    // Check if a language code was provided
     if ( newLanguageCode === '' ) {
         tgBot.sendMessage( userID, "The current language code is " +
             getLanguageCode( userID )
         );
 
+        // TODO - what if the language code is invalid?
+
         return;
     }
-
     setLanguageCode( userID, newLanguageCode );
 } );
 
+// Return the language code, if available
 var getLanguageCode = function ( userID ) {
     var languageCode;
 
+    // Check if a user with that ID is in the object of users
     if ( userStatus[userID] === undefined ) {
+        // If not, add it, and don't return anything
         userStatus[userID] = {};
-
         return '';
     }
 
+    // Else get the language code and return it
     languageCode = userStatus[userID]['languageCode'];
-
     return languageCode;
 };
 
+// Sets language code to grab untranslated messages in
 var setLanguageCode = function ( userID, newLanguageCode ) {
     debug( userID, "in setLanguageCode(), setting to " +
         newLanguageCode
     );
 
+    // Check if a user with that ID is in the object of users
     if ( userStatus[userID] === undefined ) {
         userStatus[userID] = {};
     }
 
+    // Change language code, empty messages, reset index
     userStatus[userID]['languageCode'] = newLanguageCode;
     userStatus[userID]['currentMwMessageIndex'] = 0;
     userStatus[userID]['messages'] = [];
 
+    // TODO - Maybe consider autofetching messages in new language?
```

Comment:
Yeah, probably. That's what the site essentially does when you select a new language.